Readme.md

```diff
@@ -40,6 +41,7 @@ Then list your Homebrew based dependencies in your `Brewfile`:
     brew 'mysql'
     cask 'google-chrome'
     cask 'java' unless system '/usr/libexec/java_home --failfast'
+    cask 'firefox', args: {'appdir=/Applications'}
```

Comment:
Sorry but you still fail to get it right. It should be: `{ appdir: '/Applications' }`